HHS/Head-Start-TTADP
# [TTAHUB-712] Create Script to Update Missing Creator Roles

## Description of change

Create Script to Update Missing Creator Roles.

## How to test

Run the yarn script 'updateCreatorRoles'. 

A total of 587 reports should be updated as specified in JIRA TTAHUB-609.

Spot check a few report creator roles to ensure they have been updated correctly. Viewing these reports and exporting should be without error.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-712


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
